ctapipe/containers.py

```diff
@@ -840,16 +841,29 @@ class ParticleClassificationContainer(Container):
     telescopes = Field(None, "Telescopes used if stereo, or None if Mono")
 
 
+class DispContainer(Container):
+    """
+    Standard output of disp reconstruction algorithms for origin reconstruction
+    """
+
+    default_prefix = ""
+
+    norm = Field(nan * u.deg, "reconstructed absolute value for disp", unit=u.deg)
```

Comment:
Shouldn't this be called "disp"? Norm is a bit generic.  Or instead it could be better to use "disp" as the `default_prefix`, then the output columns will be "disp_norm" and "disp_sign"

Reply:
I think using the prefix is a good idea

Reply:
Agreed. Will change that.